gammapy/irf/core.py

```diff
@@ -422,6 +422,8 @@ def to_table(self, format="gadf-dl3"):
             table.meta["HDUCLAS2"] = spec["hduclas2"]
             if self.is_pointlike:
                 table.meta["HDUCLAS3"] = "POINT-LIKE"
+            if "is_pointlike" in table.meta:
```

Comment:
This line seems superflous

Reply:
You could also  just use `table.meta.pop("is_pointlike", None)`, where the `None` suppresses the `KeyError` if the keyword is not present.

Reply:
Ah great, I didn't know this!